graylog2-server/src/main/java/org/graylog2/shared/utilities/ExceptionUtils.java

```diff
@@ -50,7 +50,7 @@ public static String formatMessageCause(Throwable t) {
             causeMessage.append(message);
         }
 
-        if (message != null && !message.endsWith(".")) {
+        if (message != null && !message.endsWith(".") && !message.endsWith("!")) {
```

Comment:
This has been fixed in https://github.com/Graylog2/graylog2-server/pull/6958/files already. Please rebase or merge master. Thanks!